src/platform/vscode-path/utils.ts

```diff
@@ -1,11 +1,5 @@
 // Miscellaneous functions from other spots in VS code
 
-import { Uri } from 'vscode';
-
-export function fsPathToUri(path: string | undefined) {
-    return path ? Uri.file(path) : undefined;
```

Comment:
Removed this as this just compilcates the code unencessarily by adding nullables to the return values, even when the path is not undefined.

I.e. if we call `fsPathToUri(<some string value>)` the return value is still nullable, when this is impossible.
This unnecessary nullable complicates the code, hence removing it.